site/docs/guides/introduction.md

```diff
@@ -0,0 +1,105 @@
+# Introduction
+
+Nessie is an OSS service and libraries that enable you to maintain multiple versions 
+of your data and leverage Git-like Branches & Tags for your Data Lake. Nessie enhances the following 
+table formats with version control techniques:
+
+* Apache Iceberg Tables ([more](../iceberg/iceberg.md))
+* Apache Iceberg Views ([more](../iceberg/views.md))
+
+## Basic Concepts
+
+Nessie is heavily inspired by Git. The main concepts Nessie exposes map directly to 
+[Git concepts](https://git-scm.com/book/en/v2). In most cases, you simply need to replace 
+references of files and directories in Git with Tables in Nessie. The primary concepts in Nessie are:
+ 
+* Commit: Consistent snapshot of all tables at a particular point in time.
+* Branch: Human-friendly reference that a user can add commits to.
+* Tag: Human-friendly reference that points to a particular commit.
+* Hash: Hexadecimal string representation of a particular commit.
+
+Out of the box, Nessie starts with a single branch called `main` that points to the 
+beginning of time. A user can immediately start adding tables to that branch. For example 
+(in pseudocode):
+
+```
+$ create t1
+...
+$ insert 2 records into t1
+...
+$ create t2
+...
+$ insert 2 records into t2
+...
+```
+
+A user can then use the Nessie CLI to view the history of the main branch. You'll see 
```

Comment:
Is this referring to Pynessie? Why not use the new CLI for these examples?

Reply:
It's not new (moved from index.md) - let's defer this to another PR.